<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup1</class>
 <widget class="QWidget" name="popup1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>181</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="coatButton">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(255, 77, 255);</string>
      </property>
      <property name="text">
       <string>Outerwear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="topButton">
      <property name="text">
       <string>Top</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bottomButton">
      <property name="text">
       <string>Bottom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shoesButton">
      <property name="text">
       <string>Shoes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
